compiler/src/dotty/tools/dotc/tastyreflect/SymbolOpsImpl.scala

```diff
@@ -181,8 +189,8 @@ trait SymbolOpsImpl extends scala.tasty.reflect.SymbolOps with CoreImpl {
   def ValSymbolDeco(symbol: ValSymbol): ValSymbolAPI = new ValSymbolAPI {
     def tree(implicit ctx: Context): ValDef = FromSymbol.valDefFromSym(symbol)
 
-    def companionClass(implicit ctx: Context): Option[ClassSymbol] = {
-      val sym = symbol.companionClass
```

Comment:
Companion class and module class are two different things

Reply:
Noted. Will fix.